Testing/SystemTests/tests/analysis/ISIS_PowderPolarisTest.py

```diff
@@ -190,7 +190,7 @@ def validate(self):
         # Whilst total scattering is in development, the validation will avoid using reference files as they will have
         # to be updated very frequently. In the meantime, the expected peak in the PDF at ~3.9 Angstrom will be checked.
         # After rebin this is at X index 37
-        self.assertAlmostEqual(self.pdf_output.dataY(0)[37], 0.8055205, places=3)
+        self.assertAlmostEqual(self.pdf_output.dataY(0)[37], 0.7376667, places=3)
```

Comment:
Could you help me understand the reason that this had to change?

Reply:
From Helen saying that we should not match spectra using the scale calulation I changed the MatchSpectra in the merge algorithm. This changed the result from the test.

Reply:
Ah yes of course. Thanks.